RunningOrder/Sprint/SprintDetail.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SprintDetail.swift
+//  RunningOrder
+//
+//  Created by Lucas Barbero on 05/08/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+struct SprintDetail: View {
+
+    var sprint: Sprint
```

Comment:
let ? ou binding ?

Reply:
since you have a viewModel, the sprint should be in it

Reply:
use viewModel.sprint everywhere x)

Reply:
Since we can't edit a sprint for now we can just use a let